website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,99 @@
+﻿
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when
+existing nodes get full, pods are stuck pending.

```

Comment:
suggest: "when existing nodes don't have available capacity, new Pods end up in a Pending state.